package.json

```
@@ -25,8 +36,7 @@
     "@reason-native/console": "*",
     "@reason-native/pastel": "*",
     "@reason-native/rely": "^3.2.1",
-    "esy-libpng":
-    "dmtrKovalenko/esy-libpng#a8f4da71d87e5bfbf567f472c0e52c95db90e04d",
+    "esy-libpng": "dmtrKovalenko/esy-libpng#4895961b8e92ff0ebba78df6ccb338ecd2272003",
```

```suggestion
    "esy-libpng": "dmtrKovalenko/esy-libpng#a8f4da71d87e5bfbf567f472c0e52c95db90e04d",
```

4895961b8e92ff0ebba78df6ccb338ecd2272003